gef-ui/server/utils/facility-helpers.js

```diff
@@ -96,18 +96,38 @@ const areUnissuedFacilitiesPresent = (application) => {
   return hasUnissuedFacilities;
 };
 
+// adds 3 months to date and returns formatted string deadline date
+const formatIssueDeadlineDate = (initialDate) => {
+  // if initialDate then uses that or if not provided then uses today
+  const date = initialDate || new Date();
+  const deadlineDate = add(date, { months: 3 });
+
+  return format(deadlineDate, 'dd MMM yyyy');
+};
+
 /*
    This function sets the deadline to display for unissued
    facilities on the unissued facilities table 3 months
-   from date of submission
+   from date of submission or MIN submission or now based on submission type
 */
-const facilityIssueDeadline = (submissionDate) => {
-  if (submissionDate) {
-  // converts to timestamp from epoch - '+' to convert from str to int
-    const date = new Date(parseInt(submissionDate, 10));
-    const deadlineDate = add(new Date(date), { months: 3 });
+const facilityIssueDeadline = (application) => {
+  // if MIA, then deadline should be 3 months from today
+  if (application.submissionType === CONSTANTS.DEAL_SUBMISSION_TYPE.MIA) {
+    return formatIssueDeadlineDate();
```

Comment:
can we pass the new Date here? this to keep it consistent with the other calls on line 123 and 130

Reply:
@abhi-markan suggested removing it and keeping the check if exists.  Which should I do?